backend/packages/identity-scan/src/scan/events/identity/index.js

```diff
@@ -1,15 +1,66 @@
-async function handleIdentityEvents(
-  event,
-  indexer,
-  extrinsic,
-  blockEvents = [],
-) {
+const { setIdentity, deleteIdentity } = require("./identityOperations");
+const { setRegistrarJudgement } = require("./registrarOperations");
+
+const {
+  IDENTITY_SET,
+  IDENTITY_CLEARED,
+  IDENTITY_KILLED,
+  JUDGEMENT_GIVEN,
+  JUDGEMENT_REQUESTED,
+  JUDGEMENT_UNREQUESTED,
+  SUB_IDENTITY_ADDED,
+  SUB_IDENTITY_REMOVED,
+  SUB_IDENTITY_REVOKED,
+  IDENTITY,
+} = require("./constants");
+const {
+  setSubIdentity,
+  deleteSubIdentity,
+} = require("./subIdentityOperations");
+const { setIdentityEventForTimeline } = require("./identityTimelineOperations");
+
+/**
+ * Handle identity events and save to DB
+ *
+ * @param event
+ * @param indexer
+ * @param extrinsic
+ * @param blockEvents
+ * @returns {Promise<void>}
+ */
+async function handleIdentityEvents(event, indexer) {
   const { section, method } = event;
-  if ("identity" === section) {
-    return;
-  }
 
-  // todo: handle various identity events, extract business data and save it to DB
+  if (IDENTITY === section) {
```

Comment:
It's better to return early.  Following code is preferred. So we don't have to wrap big logic in a nested block.

```javascript
if (IDENTITY !== section) {
  return
}
```